tests/test_checks.py

```diff
@@ -0,0 +1,142 @@
+from django.test import SimpleTestCase
+from django.test.utils import override_settings
+from django.core import checks
```

Comment:
you don't need to run the `export` step - `tox` strips all environment variables before invocation so it doesn't do anything for you. Our `runtests.py` script sets the correct environment variable for the settings to get used.

The problem here is with the packaging - `setup.py` is importing `corsheaders` to get the version number, but this is done *before* django is installed. `setup.py` really shouldn't import the package it's for, but instead use parsing code to read the version out, like https://github.com/adamchainz/django-mysql/blob/master/setup.py#L14 . I'll update this on `master` in another pull request, then you can rebase when I've done so 👍